analyses/malawi/notebooks/mwi_ecmwf_monthly_skill_dryspells.md

```diff
@@ -521,9 +522,9 @@ df_pr_sep_lt_m_sel=df_pr_sep_lt_m[(df_pr_sep_lt_m.threshold>=160)&(df_pr_sep_lt_
 ```
 
 ```python
-threshold_perc=df_pr_th[df_pr_th.month_ds>=df_pr_th.month_no_ds].head(1).threshold.values[0]
+# threshold_perc=df_pr_th[df_pr_th.month_ds>=df_pr_th.month_no_ds].head(1).threshold.values[0]
```

Comment:
did you want to remove commented out lines?

Reply:
No, but now added an explanation on why